doc/enterprise/auth/auth.md

```diff
@@ -0,0 +1,237 @@
+# Understanding Pachyderm access controls
+
+If access controls are activated, each data repository, or repo,
+in Pachyderm has an Access Control List (ACL) associated with it.
+The ACL includes:
+
+- `READERs` - users who can read the data versioned in the repo.
+- `WRITERs` - users with `READER` access who can also submit
+additions, deletions, or modifications of data into the repo.
+- `OWNERs` - users with READER and WRITER access who can also
+modify the repo's ACL.
+
+Pachyderm defines the following account types:
+
+* **GitHub user** is a user account that is associated with
+a GitHub account and logs in through the GitHub OAuth flow. If you do not
+use any third-party identity provider, you use this option. When a user tries
+to log in with a GitHub account, the system verifies the identity and
```

Comment:
minor nit: "the system" here refers to pachd (that's what verifies the identity and sends the pachyderm token), whereas under **SAML user** below, "the system" refers to the SAML ID provider (that's what confirms the identity, and responds with the SAML IdP token)

Very minor, but do you think would it make sense to change one? E.g. use "pachd" here and "the system" below or some such?